es-models/case_centric/case_centric.mapping.yaml

```diff
@@ -329,6 +326,15 @@ properties:
         properties:
           chromosome:
             type: keyword
+          clinical_annotations:
+            properties:
+              civic:
+                properties:
+                  gene_id:
+                    type: keyword
+                  variant_id:
+                    type: keyword
+            type: nested
```

Comment:
Does `clinical_annotations` need to be of type `nested`? That would only be necessary if we needed to store multiple CIVIC annotations for the same SSM.